index.html

```diff
@@ -69,17 +70,36 @@ <h2>Participate</h2>
 							  Your contribution will go directly towards buying a full page ad in the Wall Street Journal. This generally costs $350,000 USD. We want your help to get this wizard into this old newspaper. If you participate, you will not get your money back. Unless we fail. Then you will get your money back. If we succeed, we'll get the Bitcoin Wizard into the Wall Street Journal.
 							</p>
 							<p>
-							  Pay to <a href="https://insight.bitpay.com/address/3LPbF1wufpLzRU5pBNutivadVXyK9jwtkP" >3LPbF1wufpLzRU5pBNutivadVXyK9jwtkP</a> to contribute  
+							  Pay to <a href="https://insight.bitpay.com/address/3LPbF1wufpLzRU5pBNutivadVXyK9jwtkP" >3LPbF1wufpLzRU5pBNutivadVXyK9jwtkP</a> to contribute
 							</p>
+							<div class="row twitter btn-group">
+								<a href="https://twitter.com/intent/tweet?text=Just%20Contributed%20to%20get%20BTC%20Wizard%20on%20the%20Wall%20Street%20Journal!%20Join%20our%20ITO%20at&url=https%3A%2F%2Fbtcwizard.fund%2F&hashtags=BTCWiz,ITO,CryptoWSJ" target="_blank">
+								<button type="button" class="btn btn-primary">
+										Tweet this
+										<i class="fa fa-twitter"></i>
+								</button>
+								</a>
+								<a href="https://twitter.com/intent/tweet?text=Just%20Contributed%20to%20get%20BTC%20Wizard%20on%20the%20Wall%20Street%20Journal!%20Join%20our%20ITO%20at&url=https%3A%2F%2Fbtcwizard.fund%2F&hashtags=BTCWiz,ITO,CryptoWSJ&in-reply-to=930933869066338304" target="_blank">
+								<button type="button" class="btn btn-primary">
+										Join the conversation
+										<i class="fa fa-twitter"></i>
+								</button>
+								</a>
+
+							</div>
 						</div>
 						<div class="image">
 						  <a href="https://insight.bitpay.com/address/3LPbF1wufpLzRU5pBNutivadVXyK9jwtkP" target="blank">
 							<img src="images/3LPbF1wufpLzRU5pBNutivadVXyK9jwtkP.png" alt="" />
 						  </a>
 						</div>
+

```

Comment:
Try to remove these so that the only lines that are changed, are the ones that we intend to change